### PR TITLE
Theming - UI Semantic

### PR DESCRIPTION
**Pull request by soheimam**
Removed all previous CSS libraries. 
Implemented the new semantic UI 

You can find additional information  on components here - https://react.semantic-ui.com/
When creating buttons and needing them to be specific colours  we can add ClassName=' ' ;

List of class names for additional colors can be found in App.css.

**For testing** 

For testing purposes if you could see that when creating a new button like so - <Button>Click Here</Button>
The button by default comes in with the correct branding blue.  Creating titles  information can be found here under page headers.

- -> example components can be found here - https://master.d2rb7w0g8t39tq.amplifyapp.com/
-  -> https://react.semantic-ui.com/elements/header/
- -> for form input components https://react.semantic-ui.com/modules/checkbox/

### CODE REVIEW

**Comment by DurkoMatko** [comment]
Will take a look tonight :)

**Comment by mcgnly** [comment]
I'm running it now and the sign-in and log-out button both have classes that to me would imply that they're blue, but they show up as grey to me, after running yarn install. Do I need to run some build step for it to work?

File: src/views/TabBar.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {
   faCogs,
   faQrcode,
 } from "@fortawesome/free-solid-svg-icons"
-import "../App.scss"
+// import "../App.scss"
```
Comment (DurkoMatko) [inline src/views/TabBar.tsx]: this is imported in index.js now right? so can be deleted from here?

File: src/views/NavMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  
+  const options = [
+    {
+      key: 'Nea Kavala',
+      text: 'Nea Kavala',
+      value: 'Nea Kavala',
+      content: 'Nea Kavala',
+    },
+    {
+      key: 'Elsfina',
```
Comment (DurkoMatko) [inline src/views/NavMenu.tsx]: It's not important at all but JFYI, it's Elefsina :)

File: src/views/ScanBox.tsx [hunk 3]
```diff
@@ -51,14 +52,22 @@ function ScanBox() {
...
+          <Button.Content hidden>
+            <Icon name='arrow left' />
+        </Button.Content>
+        </Button>
       </Link>
     </div>
   )
 }
 
+
```
Comment (DurkoMatko) [inline src/views/ScanBox.tsx]: do we need this whitespace?

**Review by DurkoMatko — APPROVED** [review]
Holy moly, it looks sooo goood :heart_eyes: I'd just remove couple of comments but it looks fine. Also, I have the same issue as @mcgnly commented with the logout button not looking as expected. But sign-in looks fine.

Btw I just merged some rules and pre-commit hook (to check semicolons, formatting and stuff like that) to master yesterday so there r some conflicts now for this PR. I'm sorry for that ... It's possible that when you resolve them and will want to commit, it will not be possible because of the linter errors. Just run `yarn format:write` and it should automatically fix your files to the latest settings :) You can actually always do that from now on when you'll be trying to commit and will get errors.

**Comment by soheimam** [comment]
Hi @DurkoMatko thank you for adding prettier, I have fixed the issues  and it is now ready for merging. @mcgnly thank you for reviewing the button problem should be fixed now!

**Pull request merged**